game/scripts/vscripts/abilities/electrician/electrician_static_grip.lua

```diff
@@ -137,14 +135,11 @@ function modifier_electrician_static_grip:OnCreated( event )
     -- grab ability specials
     local damageInterval = spell:GetSpecialValueFor( "damage_interval" )
     local damage_per_second = spell:GetSpecialValueFor("damage_per_second")
-    -- Bonus damage talent
-    local talent = caster:FindAbilityByName("special_bonus_electrician_static_grip_damage")
-    if talent and talent:GetLevel() > 0 then
-      damage_per_second = spell:GetSpecialValueFor("damage_per_second") * talent:GetSpecialValueFor("value")
-    end
```

Comment:
this is really why i like the new style talents so much, no need to specially code the implementations of the talent modified values